docs/config-schema.md

```diff
@@ -149,7 +150,8 @@ The **nested** `writer` config object has the following fields:
 | `logDimensionUpdates` | no | bool | If `true`, dimension updates will be logged at the INFO level. (**default:** `false`) |
 | `logDroppedDatapoints` | no | bool | If true, and the log level is `debug`, filtered out datapoints will be logged. (**default:** `false`) |
 | `addGlobalDimensionsAsSpanTags` | no | bool | If true, the dimensions specified in the top-level `globalDimensions` configuration will be added to the tag set of all spans that are emitted by the writer.  If this is false, only the "host id" dimensions such as `host`, `AwsUniqueId`, etc. are added to the span tags. (**default:** `false`) |
-| `sendTraceHostCorrelationMetrics` | no | bool | Whether to send host correlation metrics to correlation traced services with the underlying host (**default:** `true`) |
+| `traceInfrastructureCorrelation` | no | bool | Whether to send host correlation properties to correlate traced services with the infrastructure (**default:** `"false"`) |
```

Comment:
Just make it always send them if somebody is sending traces through the agent and take out this config option.  There are too many config options and this is way too confusing what should be on or off.  Just make it always on.